restcomm/restcomm.testsuite/src/test/java/org/mobicents/servlet/restcomm/http/AccountsEndpointTest.java

```diff
@@ -77,11 +81,29 @@ public void testCreateAccount() {
     }
 
     @Test
+    public void testCreateAccountCheckClientExisted() throws ClientProtocolException, IOException {
+        String subAccountPassword = "mynewpassword";
+        String subAccountEmail = "lyhungthinh@gmail.com";
+
+        if (!accountUpdated){
+            RestcommAccountsTool.getInstance().updateAccount(deploymentUrl.toString(), adminUsername, adminAuthToken,
+                    adminUsername, newAdminPassword, adminAccountSid, null);
+        }
+        JsonObject subAccountResponse = RestcommAccountsTool.getInstance().createAccount(deploymentUrl.toString(),
+                adminUsername, newAdminAuthToken, subAccountEmail, subAccountPassword);
+
+        JsonObject clientOfAccount = CreateClientsTool.getInstance()
+                                      .getClientOfAccount(deploymentUrl.toString(), subAccountResponse);
+        assertNotNull(clientOfAccount);
```

Comment:
After you verify that the client is created, you need to proceed to SIP Register the client with Restcomm